### PR TITLE
modelColor can be changed

### PR DESCRIPTION
**Pull request by hdriqi**
By giving next modelColor props, the model color of the STL file can be changed dynamically.

### CODE REVIEW

File: dist/STLViewer.js [hunk 1]
```diff
@@ -168,12 +169,143 @@ var STLViewer = function (_Component) {
...
+        loader.load(url, function (geometry) {
+
+          // Calculate mesh noramls for MeshLambertMaterial.
+          geometry.computeFaceNormals();
+          geometry.computeVertexNormals();
+
+          // Center the object
+          geometry.center();
+
+          console.log(modelColor);
```
Comment (chiedo) [inline dist/STLViewer.js]: Can you please remove this line @hdriqi ?

File: dist/STLViewer.js [hunk 2]
```diff
@@ -168,12 +169,143 @@ var STLViewer = function (_Component) {
       };
     }
   }, {
+    key: 'componentWillUpdate',
```
Comment (chiedo) [inline dist/STLViewer.js]: This seems like a lot of code to change the background color.

Is there anything else going on here @hdriqi ?

File: dist/STLViewer.js [hunk 3]
```diff
@@ -80,6 +80,7 @@ var STLViewer = function (_Component) {
         scene.add(directionalLight);
 
         var loader = new _Three2.default.STLLoader();
+        loader.crossOrigin = '';
```
Comment (chiedo) [inline dist/STLViewer.js]: Can you explain why you updated this @hdriqi ?

**Comment by chiedo** [comment]
Thanks for the contribution @hdriqi :)

See my comments

File: dist/STLViewer.js (hunk 2 above)
Comment (hdriqi) [inline dist/STLViewer.js]: this code is changing the model color. So, when you give stl-viewer new color props, the model color's gonna change.
My approach is just replace the old model with the new model.

File: dist/STLViewer.js (hunk 1 above)
Comment (hdriqi) [inline dist/STLViewer.js]: only the console.log?

File: dist/STLViewer.js (hunk 3 above)
Comment (hdriqi) [inline dist/STLViewer.js]: that part to make sure that stl-viewer can load any stl files from given url ( no cross origin )

**Comment by chiedo** [comment]
Hey @hdriqi can you please squash your commits?

**Comment by hdriqi** [comment]
@chiedo done

**Comment by chiedo** [comment]
Thanks @hdriqi I just pushed your changes to npm. I had to rebase because you committed the dist folder. I just added that to the README to prevent people from committing the dist folder in the future.

**Pull request closed without merging**